text/2019-11-01-tikv-community-reorg-sig.md

```diff
@@ -0,0 +1,148 @@
+
+- **Maintainer**: a planner and designer of the project; capable of merging master branches; appointed by existing Maintainers from Committers.
+- **Committer**: nominated by Maintainers or PMC as a contributor with outstanding contribution to TiKV; must have implemented at least one feature or fixed a major bug independently.
+- **Reviewer**: an advanced contributor with the approval right for code reviews; can participate in the design of subsequent projects.
+- **Active Contributor**: an active contributor to the community voted by at least two reviewers. merged at least 8 PRs
+- **Contributor**: a community contributor that has at least one PR (Pull Request) merged to any repository under the TiKV organization.
+
+Besides the Developer Group, the community also includes the following two organizations:
+
+- Organizer Committee of the project consists of organizers in charge of event or activity operation that promotes the project. Specifically, the members are the leaders of user groups or community event organizers across regions.
+- Project Management Committee (PMC) functions as the core management team of the project; decides the roadmap and makes major decisions related to the community.
+
+#### Special Interest Group
+
+![Special Interest Group](../media/tikv-developer-group.png)
+
+The above diagram shows the overall structure of the developer community and reflecting the second change in this community architecture upgrade - the introduction of the Special Interest Group (SIG).
+
+An SIG mainly leads the development and maintenance of a TiKV module and is responsible for the quality of the module's code. Active Contributors that meet the requirements will be invited to an SIG. Developers in the SIG will receive continuous guidance from Tech Leads while they optimize and improve the module, and meanwhile, exercise their technical skills. Community developers can gradually grow from the initial Active Contributor to the community-recognized Reviewer, Committer, and even Maintainer through SIGs. In general, each SIG will organize meetings periodically to discuss about the recent progress and the problems encountered. All meeting discussions are open to the community, so that interested community members can participate.
+
+In addition, this picture also reflects the relationship between community roles and SIGs.
+
+- Active Contributor
+    + Must be a Contributor with at least 8 PRs merged to the master branch within 1 year.
+    + To join an SIG, a Contributor needs to contribute more than 8 PRs to the module that the SIG is responsible for within 1 year, so that he/she can be invited to join the SIG for targeted learning and contributions.
+
+- Reviewer
+    + Must belong to a SIG; has the obligation to review PRs.
+  + Generated from Active Contributors. When an Active Contributor has significant contributions to the module and is nominated by 2 or more Committers, he/she will be invited to be a Reviewer of the module.
```

Comment:
This line has different indention on my computer.

Reply:
Fixed.